shared-ts/disruptionTypes.zod.ts

```diff
@@ -760,8 +761,36 @@ const baseConsequence = {
 export const networkConsequenceSchema = z.object({
     ...baseConsequence,
     consequenceType: z.literal("networkWide", setZodDefaultError("Select a consequence type")),
+    disruptionArea: z
+        .preprocess(
+            (val) => transformToArray(val),
+            z
+                .array(z.string(setZodDefaultError("Select one or more disruption areas")))
+                .min(1, { message: "Select one or more disruption areas" })
+                .optional(),
```

Comment:
I think we can remove this optional() since we have it at the top level now
```suggestion
      
```

Reply:
done